js/contributors.js

```diff
@@ -55,7 +55,7 @@ $(document).ready(function(){
       $(".issue-container").append("<li class='mi'>Sorry there are no issues labeled feature currently opened at gci17.fossasia.org</li>");
     }
 
-    $(".issue-container").append("<a href='https://github.com/fossasia/gci17.fossasia.org/issues' class='btn totalIssue'>" + allIssues.length + " issues on <i class='fa fa-github'></i> Github");
+    $(".issue-container").append("<a href='https://github.com/fossasia/gci17.fossasia.org/issues?utf8=%E2%9C%93&q=is%3Aopen+' class='btn totalIssue'>" + allIssues.length + " activities on <i class='fa fa-github'></i> Github");
```

Comment:
Let the link remain same: `href=https://github.com/fossasia/gci17.fossasia.org/issues`

Reply:
But then, it only shows the issues and not all activities.

Reply:
All right, I'll change that.

Reply:
@harshit98 Done!